### PR TITLE
nixpkgs: -> release-16.03

### PR DESCRIPTION
**Pull request by fmap**
Onwards!

### CODE REVIEW

File: pkgs/mergex/default.nix [hunk 1]
```diff
@@ -1,10 +1,11 @@
-{ stdenv, fetchsvn }:
+{ stdenv, fetchgit }:
 
 stdenv.mkDerivation {
   name = "mergex";
-  src = fetchsvn {
-    url = "http://mergex.googlecode.com/svn/trunk";
```
Comment (ip1981) [inline pkgs/mergex/default.nix:6]: lol

File: nixos/default.nix [hunk 2]
```diff
@@ -44,10 +38,12 @@ let
       powerManagement = stub;
       security.pam.usb = stub;
       security.pam.mount = stub;
+      security.pam.oath = stub;
       boot.isContainer = lib.mkOption { default = false; };
       boot.initrd.luks = stub;
       networking.wireless = stub;
       networking.connman = stub;
+      virtualisation.vswitch = stub;
```
Comment (phunehehe) [inline nixos/default.nix:46]: Why do we need these stubs? If I'm reading this correctly they disable the services. Does that imply these services are enabled by default and we want to turn them off?

File: pkgs/curator/default.nix [hunk 3]
```diff
@@ -9,5 +9,7 @@ buildPythonPackage {
     sha256 = "1p6qmfk9j9ng8rgkz8q3mc7f9mc96xhn585g14f8zqy65wflz3q1";
   };
 
-  propagatedBuildInputs = [click elasticsearch urllib3 nosexcover mock];
+  doCheck = false;
```
Comment (phunehehe) [inline pkgs/curator/default.nix:14]: Why do we not want check? Is it too slow or does it do something weird?

**Comment by phunehehe** [comment]
the rest is nice (and yay for negative code)

File: nixos/default.nix (hunk 2 above)
Comment (fmap) [inline nixos/default.nix:46]: These're options referenced by the proper subset of modules we bring in via `vendor-modules.nix`; we want to keep their references intact to support evaluation, but for some reason (smaller closures? historical misbehaviour?) have no behaviour beyond placeholding. In this case, the `network-interfaces` module will set `vswitch` config, under certain configuration, but we never use those, so.

File: pkgs/curator/default.nix (hunk 3 above)
Comment (fmap) [inline pkgs/curator/default.nix:14]: IIRC one of the test packages had tricky version bounds, and I didn't feel like re-packaging it.

Comment (phunehehe) [inline pkgs/curator/default.nix:14]: can you add that as a comment then? would be useful if we ever want to revisit.

**Comment by phunehehe** [comment]
Here's my take on a roll out plan:

 - merge this now
 - it goes to staging next Wed (06/21)
   - if there is no problem, it goes to live on 06/29
   - if there are problems, bump staging to the commit before this, fix problems and try again each week

@fmap it seems suitable for me to take over, now that it involves arranging between teams. Let me know if you have any concern. If you don't, merge this and assign https://github.com/zalora/eris/issues/1838 to me.

**Pull request merged**